src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,60 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
     public static void main(String[] args) {
-        // test your code here
+        System.out.println("\n-----------------\nCheck creating new car");
+        Manufacturer manufacturer = new Manufacturer(0L, "ZAZ", "Ukraine");
```

Comment:
We should not put `id` to the constructor. `id` will be generated by DB and returned in result of  `manufacturerService.create()` method.
Create constructor without `id` field.
Fix the same suggestion for other model classes.